homeassistant/components/august/__init__.py

```diff
@@ -284,58 +294,51 @@ def get_door_state(self, lock_id):
 
         This is the status from the door sensor.
         """
-        self._update_doors()
+        self._update_locks_status()
         return self._door_state_by_id.get(lock_id)
 
+    def _update_locks(self):
+        self._update_locks_status()
+        self._update_locks_detail()
+
     @Throttle(MIN_TIME_BETWEEN_UPDATES)
-    def _update_doors(self):
+    def _update_locks_status(self):
+        status_by_id = {}
         state_by_id = {}
 
-        _LOGGER.debug("Start retrieving door status")
+        _LOGGER.debug("Start retrieving lock and door status")
         for lock in self._locks:
-            _LOGGER.debug("Updating door status for %s", lock.device_name)
-
+            _LOGGER.debug("Updating lock and door status for %s", lock.device_name)
             try:
-                state_by_id[lock.device_id] = self._api.get_lock_door_status(
-                    self._access_token, lock.device_id
+                (
+                    status_by_id[lock.device_id],
+                    state_by_id[lock.device_id],
+                ) = self._api.get_lock_status(
+                    self._access_token, lock.device_id, door_status=True
                 )
             except RequestException as ex:
                 _LOGGER.error(
-                    "Request error trying to retrieve door status for %s. %s",
+                    "Request error trying to retrieve locka dn door status for %s. %s",
```

Comment:
looks like a small typo

```suggestion
                    "Request error trying to retrieve lock and door status for %s. %s",
```